src/components/logs-page/index.tsx

```diff
@@ -125,6 +126,28 @@ class LogsPage extends Component<PropsFromStore & BridgeApi & UtilsApi & WithTra
                                 onChange={updateBridgeConfig}
                             />
                         </div>
+                        <div className="col-12 col-sm-4 col-xxl-4">
+                            <label htmlFor="logs-limit" className="form-label">
+                                {t('logs_limit')}
+                            </label>
+                            <select
+                                id="logs-limit"
+                                className="form-select"
+                                onChange={(e) => {
+                                    const limit = parseInt(e.target.value);
+                                    store.setState({
+                                        logsLimit: limit,
+                                        logs: [...store.getState().logs.slice(-limit)],
```

Comment:
I don't think we should slice logs here, this will happen automatically once new log record arrives, and then handler can be simplified to 
`store.setState({logsLimit: parseInt(e.target.value)})`

Reply:
> this will happen automatically once new log record arrives

yep, but I think it's better for the user experience, if the slice is also done immediately upon selecting the option from the menu

it doesn't matter for 100 -> 1000 change, but it does in the case of changing 1000 -> 100 ;)

i don't think the extra slicing complicates this code so much, to make a worse UI instead…